pkgs/tools/filesystems/ceph/arrow-cpp-19.nix

```diff
@@ -101,6 +99,15 @@ stdenv.mkDerivation (finalAttrs: {
 
   sourceRoot = "${finalAttrs.src.name}/cpp";
 
+  patches = [
+    (fetchpatch2 {
+      name = "protobuf-30-compat.patch";
+      url = "https://github.com/apache/arrow/pull/46136.patch";
```

Comment:
Probably better to set this to `https://github.com/apache/arrow/compare/a3a0c16a823387537243d513ba89d853d9b5eed6~..de11a52b9f57c89f02fabcf65ec80d0fe4d54562.patch` than the PR number. Seems to have the same `hash`, so should not change the eval.

Reply:
Just checked the *CONTRIBUTING.md* and there doesn't seem to be any specific guidance on that other than not referencing *open* PRs (for obvious reasons).
Personally I prefer linking to the actual PR so it's easier to figure out where stuff came from, and since the PR is merged it can't change or be reopened AFAIK, but I'm fully okay changing this.

I'll have to clean up this PR before undrafting anyway, so I'll get back to that later (just pushed the revert in here to have a canonical branch to test against).

Reply:
@benaryorg I also think either is OK, it really is a nitpick; the nice thing about the commit based URL is that it allows to figure out what's in it even if the upstream does weird things such as reopening the PR and force-pushing the branch (not sure why somebody would want to do that htough), or deleting the whole repo (thus making it slightly less dependentent on info stored only on Github).